### PR TITLE
Bionic: Sync shell colors and buttons with master

### PR DESCRIPTION
**Pull request by Feichtmeier**
- sync shell popup with master shell popup color fafafa
- darken slider handle border colors like in master
- update button shadows like in master

### CODE REVIEW

File: gnome-shell/src/gnome-shell-sass/_common.scss [hunk 1]
```diff
@@ -153,15 +153,15 @@ StScrollBar {
   -barlevel-border-width: 1px;
   -slider-handle-radius: 8px;
   -slider-handle-border-width: 1px;
-  -slider-handle-border-color: $borders_color;
+  -slider-handle-border-color: darken($alt_borders_color, 3%);
```
Comment (clobrano) [inline gnome-shell/src/gnome-shell-sass/_common.scss]: You should import `$alt_borders_colors` in `_colors.scss`

File: gnome-shell/src/gnome-shell-sass/_common.scss [hunk 2]
```diff
@@ -153,15 +153,15 @@ StScrollBar {
...
   color: lighten($bg_color, 4%);
   &:hover { color: $_hover_bg_color; }
   &:active { color: $_active_bg_color; }
 
   /* START: VERSION SPECIFIC CODE (< 3.29.90) */
   -slider-height: 0.2em; //increase the height because of the border removal
   -slider-background-color:  mix($borders_color, $bg_color, 50%);
-  -slider-border-color: $borders_color; //trough border color
+  -slider-border-color: darken($alt_borders_color, 3%); //trough border color
```
Comment (clobrano) [inline gnome-shell/src/gnome-shell-sass/_common.scss]: same as above

Comment (Feichtmeier) [reply gnome-shell/src/gnome-shell-sass/_common.scss]: woopsie! yeah "debugging" via snap and test xD

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr2089`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr2089` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr2089`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr2089` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Pull request merged**